src/lib/config.js

```diff
@@ -116,13 +116,27 @@ function parseLedgers () {
   })
 }
 
+function parseRoutes () {
+  return JSON.parse(Config.getEnv(envPrefix, 'ROUTES') || '[]')
```

Comment:
Might be good to validate the data that's passed in, just so you throw an error immediately if someone gives invalid configuration rather than waiting until it causes a problem when the connector tries to use it